tutorials/data-lake/pull-dataset/README.md

```diff
@@ -1 +1,16 @@
-# Pull dataset from Rucio data lake
+# Interact with Rucio dataset files
+
+The following script assumes that all the files within a DID are present in a RSE (Rucio Storage Element), and that this RSE is accessible locally.
```

Comment:
It is explained in the line immediately below :) 
Also here https://rucio.github.io/documentation/started/concepts/file_dataset_container/

This is a general comment for the whole interTwin community; we can build thousands of  transparent and abstraction layers to ease the life of users, but a minimal understanding of the service is required.

Therefore I will suggest contacting with Diana to understand where is the rucio tutorial and how it can be link here, for example

Reply:
I think that adding "dataset container" plus the link above would be a good starting point. I can link the webinar as well